crates/net/network/src/session/active.rs

```diff
@@ -165,6 +173,7 @@ impl<N: NetworkPrimitives> ActiveSession<N> {
     ///
     /// Returns an error if the message is considered to be in violation of the protocol.
     fn on_incoming_message(&mut self, msg: EthMessage<N>) -> OnIncomingMessageOutcome<N> {
+        let now = Instant::now();
```

Comment:
we can avoid this here and instead only instantiate this when we need to record this based on the message variant